### DISCUSSION TITLE
[Question] Custom buy and sell prices

### DISCUSSION THREAD

**Discussion by szucsaaron2** [kernc/backtesting.py#597] [Q&A] [answered]
Hey!

First of all thank you for your amazing work! This library is 10/10!

My questions is, is that possible to somehow change the buy and open prices? 
When I am live trading, my strategy takes some time to calculate all the indicators, and buying and selling usually doesn't happen on close/next open prices. Sometimes this loading process can take up to 2 minutes. Under those 2 minutes the prices can change significantly.

I already wrote my own really simple backtesting function which uses a "worst case scenario". So buy happens on next high, and sell happens on next low. But your library is so much cooler, and I love the visualization. Would it be possible to somehow set these custom prices in backtesting.py?

Thank you and have a nice day!
szucsaaron

**Comment by kernc** [comment, marked as answer]
Most simply, increase your [`Backtest(..., commission=`)](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Backtest), which will be applied on every trade.

**Reply by szucsaaron2** [reply]
Thank you, that's a good idea!
